ceresdb-protocol/src/main/java/io/ceresdb/RouterClient.java

```diff
@@ -197,7 +200,6 @@ public CompletableFuture<Map<String, Route>> routeFor(final RequestContext reqCt
 
     public CompletableFuture<Map<String, Route>> routeRefreshFor(final RequestContext reqCtx,
                                                                  final Collection<String> tables) {
-        final long startCall = Clock.defaultClock().getTick();
```

Comment:
Why remove it? it is used for call time statistics.

Reply:
some time statistics is not used in current version